README.md

```diff
@@ -24,6 +24,10 @@ WinUI usage documentation can be found here:
 
 https://docs.microsoft.com/uwp/toolkits/winui
 
+Release notes for Microsoft.UI.Xaml WinUI packages can be found here:
+
```

Comment:
This (and the previous usage documentation link) would read better if there were no blank line between them and the text.